### PR TITLE
add 4.13 multi ec.3 dev preview images

### PR DESCRIPTION
**Pull request by jacobemery**
@eliorez added 4.13 dev preview images (#4898), and then bumped to ec.2 (#4933).

This commit adds 4.13.0 ec.2 images for ppc64le and s390x, in parity with #4933; and adds 4.12 ppc64le and s390x ISOs where they are needed in order to test 4.13 dev preview images, in parity with #4898.

## List all the issues related to this PR
#4898 
#4933
[assisted-image-service PR #116](https://github.com/openshift/assisted-image-service/pull/116)

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [x] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

/cc @eliorerz
/cc @carbonin
/cc @osherdp
/cc @gamli75 

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: data/default_os_images.json [hunk 1]
```diff
@@ -82,5 +82,17 @@
...
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
+        "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "s390x",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-s390x-live.s390x.iso",
```
Comment (eliorerz) [inline data/default_os_images.json:95]: 4.13 is available, you can use this image instead 
`https://mirror.openshift.com/pub/openshift-v4/s390x/dependencies/rhcos/pre-release/latest-4.13/rhcos-4.13.0-ec.3-s390x-live.s390x.iso`

File: data/default_os_images.json [hunk 2]
```diff
@@ -82,5 +82,17 @@
         "cpu_architecture": "arm64",
         "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-aarch64-live.aarch64.iso",
         "version": "412.86.202212081411-0"
+    },
+    {
+        "openshift_version": "4.13",
+        "cpu_architecture": "ppc64le",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/4.12.0/rhcos-4.12.0-ppc64le-live.ppc64le.iso",
```
Comment (eliorerz) [inline data/default_os_images.json:89]:
```suggestion
        "url": "https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/pre-release/latest-4.13/rhcos-4.13.0-ec.3-ppc64le-live.ppc64le.iso",
```

**Comment by gamli75** [comment]
/hold
please fix the 4.13 images

File: data/default_os_images.json (hunk 2 above)
Comment (jacobemery) [reply data/default_os_images.json]: Bumped all 4.13 dev preview images to ec.3.

Now running skipper make generate and will push once done.

Comment (jacobemery) [reply data/default_os_images.json]: Done ^

File: data/default_os_images.json (hunk 1 above)
Comment (jacobemery) [reply data/default_os_images.json]: Done!

**Comment by jacobemery** [comment]
> /hold please fix the 4.13 images

done!

**Comment by jacobemery** [comment]
Also switched to 4.13 dev preview images in the corresponding assisted-image-service [PR](https://github.com/openshift/assisted-image-service/pull/116).

**Review by osherdp — APPROVED** [review]
/unhold

**Pull request merged**